env_setup/set_default_vm_vars.yml

```diff
@@ -149,17 +140,14 @@
     cpu_cores_per_socket: "{{ guest_config_options.rec_cpu_cores_per_socket }}"
   when: cpu_number is undefined or not cpu_number
 
-- name: "Check VM CPU socket number"
+- name: "Check Windows VM CPU socket number"
   include_tasks: check_cpu_socket.yml
-
-- name: "Set fact of the VM datastore path"
-  ansible.builtin.set_fact:
-    vm_datastore_path: "/vmfs/volumes/{{ datastore }}"
+  when: testing_testcase_file | dirname | basename == "windows"
```

Comment:
We have a variable `gosv_test_suite` defined in `main.yml`, here can change to `gosv_test_suite == "windows"`